poucave/html/js/app.js

```diff
@@ -690,7 +712,12 @@ class Markdown extends Component {
 }
 
 // Initialize the app on page load
-window.addEventListener("load", () => {
+let diagramStylesheet;
+window.addEventListener("load", async () => {
+  // Load the stylesheet to be injected into the SVG
+  const diagramStylesheetResponse = await fetch("/html/css/svg-diagram.css");
+  diagramStylesheet = await diagramStylesheetResponse.text();
```

Comment:
Can we avoid the global variable approach? :) 

For example, can we do something like `<style id="svg-style">...</style>` + `document.getElementById("svg-style").textContent`?

Or can we retrieve the content of a link BTW? `<link rel="stylesheet" >`? 

Or can we put the SVG styles just as a string? Maybe we don't strokes or dark variation for dots...  What do you think?